src/styles/sidebar/components/thread.scss

```diff
@@ -0,0 +1,50 @@
+@use "../../variables" as var;
+
+.thread {
+  &--reply {
+    margin-top: 0.5em;
+    padding-top: 0.5em;
+  }
+  display: flex;
+
+  &__collapse {
+    border-left: 1px dashed var.$grey-3;
+    margin: 0.25em;
+    margin-top: 0;
+    cursor: auto;
+
+    &:hover {
+      border-left: 1px dashed var.$grey-4;
+    }
+  }
+
+  .is-collapsed &__collapse {
+    border-left: none;
+  }
+
+  &__collapse-button {
+    margin-left: -1.25em;
+    padding: 0.75em;
+    padding-top: 0.25em;
+    padding-bottom: 1em;
+    background-color: var.$white;
```

Comment:
This was necessary so that the dashed border line doesn't show through the button.